CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+### Changed
+- Updated Go versions to 1.15.
```

Comment:
You can skip the CHANGELOG.md entry for this since the Golang version isn't a user-facing change for Summon behavior. (It would be different story if this were e.g. a Golang API repository where the Golang version is important for the user to know).
https://github.com/cyberark/community/blob/master/Conjur/CONTRIBUTING.md#changelog-content

Reply:
ok, thanks I will remove it.